symphony-app-authentication-fe/services/registerApplication.js

```diff
@@ -0,0 +1,23 @@
+import { Utils } from '../js/utils.service';
+
+const dependencies = [
```

Comment:
Harcoded? The library consumer should have the ability to provide a custom list of dependencies.  Only 'extended-user-info' service is required to retrieve JWT.

Reply:
Done.